src/components/common/Dialogs/ManageWhitelistDialog/ManageWhitelistDialogForm.tsx

```diff
@@ -180,7 +184,11 @@ const ManageWhitelistDialogForm = ({
         <DialogControls
           secondaryButtonText={backButtonText}
           onSecondaryButtonClick={back}
-          disabled={!isDirty || disabledSubmit}
+          disabled={
+            (!isDirty && !whitelistAddress) ||
```

Comment:
This could probably be expressed better, but basically we want to disable the form if it isn't dirty, except for when there's an address already populated in the address field.

Reply:
Could this comment live in-code?